index.js

```diff
@@ -0,0 +1,16 @@
+import Card from './js/Card.js';
+import Create from './js/Create.js';
+import Nav from './js/Navigation.js';
+
+Nav();
+
+const allCardElements = document.querySelectorAll('[data-js="card"]');
+const allCreateElements = document.querySelectorAll('[data-js="create"]');
+
+allCardElements.forEach((cardElement) => {
```

Comment:
this i checked out this afternoon: it is the function (assigned to a variable called method i believe) `.forEach()` with the task walking through the list of all card elements saved in the variable `allCardElements`step by step to do what we write as a whole new function inside its ()        
written as an arrow function `cardElement => {Card(cardElement)};` so you don't need the normal two parenthesis around the first `cardElement`

Reply:
ah okay, i will try it out, this would have been the next step to fix my forEach/ All bug 😊
Thank you!! 😊